src/planner/vectorfield/MoveEndEffectorOffsetVectorField.cpp

```diff
@@ -50,7 +50,7 @@ MoveEndEffectorOffsetVectorField::MoveEndEffectorOffsetVectorField(
 bool MoveEndEffectorOffsetVectorField::evaluateCartesianVelocity(
     const Eigen::Isometry3d& pose, Eigen::Vector6d& cartesianVelocity) const
 {
-  using dart::math::logMap;
+  using ::dart::math::logMap;
```

Comment:
Not really a review thing but more a question from somebody who doesn't know C++ as well- what is the purpose of adding `::` here?

Reply:
The leading `::` is used to represent the following namespace is a top-level namespace. We use this when there is the same name namespace in the "active" namespace. In our case, we use it to distinguish `::dart` (i.e., the DART's top-level namespace) and `::aikido::statespace::dart`/`::aikido::planner::dart`.